scripts/lintdoc.lua

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env -S nvim -l
+
+-- Validate vimdoc files on $VIMRUNTIME/doc, and test generating HTML docs.
+-- Checks for duplicate/missing tags, parse errors, and invalid links/urls/spellings.
+-- See also `make lintdoc`.
+--
+-- Usage:
+--   $ nvim -l scripts/lintdoc.lua
+--   $ make lintdoc
+
+print('Running lintdoc ...')
+
+-- gen_help_html requires :helptags to be generated on $VIMRUNTIME/doc
+-- :helptags checks for duplicate tags.
+vim.cmd [[ helptags ALL ]]
```

Comment:
Is this still needed? If so I can take a look at the build system to fix it instead of trying to work around it.

Reply:
Yes, please---that has bitten people before.

Reply:
Good point. `:helptags` should've already done in `make doc`; `lintdoc` should *fail* instead if helptags were not generated.

Reply:
I meant that the cmake target `lintdoc` already depends on the target `nvim_runtime`, so this should not be needed. If it is then something is incorrect somewhere.